src/store.js

```diff
@@ -15,6 +15,21 @@ var store = {
     auth: false
 };
 
+
+store.resetStore = function() {
+    // TODO : I'm thinking we need some user/trader objects that manage this.
+    this.user = {
+        id: null,
+        traders : []
+    };
+    this.trader = {
+        id : null,
+        pendedVouchers : []
+    };
+    this.vouchers = this.vouchers.splice(0, this.vouchers);
```

Comment:
OK - keen not to over complicate - which is why my instinct was this.vouchers = []

So it does what it says in the name 'resetStore'

But happy with your judgement on this.